templates/zerver/api/changelog.md

```diff
@@ -15,6 +15,10 @@ below features are supported.
 * [`POST /register`](/api/register-queue): Increased
   `realm_community_topic_editing_limit_seconds` time limit value
   to 259200s (3 days).
+* [`GET /messages`](/api/get-messages) and [`GET
+  /events`](/api/get-events): `topic_links` returns a list
+  of objects each containing the `url` and `text` for links
+  that are present in the topic or generated by linkifiers.
```

Comment:
This needs a corresponding change to the documentation in `zerver/openapi/zulip.yaml`.